test/controllers/old_node_controller_test.rb

```diff
@@ -90,6 +90,9 @@ def test_version
     xml_node = xml_doc.find("//osm/node").first
     nodeid = current_nodes(:node_with_versions).id
 
+    # Ensure that the current tags are propogated to the history too
+    propogate_tags(current_nodes(:node_with_versions), nodes(:node_with_versions_v4))
```

Comment:
I've fixed the spelling, and added some tags to be propagated. The confusion is that I refactored the old_node into factories first, and then the tags were inconsistent, but after refactoring the current_nodes into factories, the tags were consistently missing!

Of course, it's a sign of dodgy tests that one can miss out the tag creation without causing test failures, but I want to avoid whole-sale rewriting of all the tests and just focus on swapping fixtures for factories.